src/test/java/org/gridsuite/ds/server/controller/utils/ParameterUtils.java

```diff
@@ -181,8 +181,15 @@ public static List<EventInfos> getEventInfosList() {
         return List.of(
                 new EventInfos(null, null, "_BUS____1-BUS____5-1_AC", null, "Disconnect", List.of(
                         new EventPropertyInfos(null, "staticId", "_BUS____1-BUS____5-1_AC", PropertyType.STRING),
```

Comment:
```suggestion
                        new EventPropertyInfos(null, FIELD_STATIC_ID, "_BUS____1-BUS____5-1_AC", PropertyType.STRING),
```

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/95/commits/3f2f55b576f7e0b3ade71c709a55bcbdd3572e9e